Dockerfile.ppc64le

```diff
@@ -0,0 +1,19 @@
+FROM mambaorg/micromamba
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+USER root
+
+RUN apt-get update  -y     && apt-get install -y git wget vim numactl gcc-12 g++-12 protobuf-compiler libprotobuf-dev     && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-12 10 --slave /usr/bin/g++ g++ /usr/bin/g++-12
+
+RUN micromamba install -y -n base -c rocketce  -c defaults     python=3.10     pytorch-cpu     numba     llvmlite     httptools &&     micromamba clean --all --yes
```

Comment:
Hi @mgoin the reason for using mamba/conda based builds is that IBM provides optimized packages for ppc64le processors by the [open-ce project](https://github.com/open-ce/open-ce) which are then built and hosted by universities like [Oregon State University](https://ftp.osuosl.org/pub/open-ce/1.11.0-p10/) or partners like [RocketCE](https://anaconda.org/rocketce).

The issue with venv & pip is that many open-source project do not provide pre-built wheels for Power architecture and thus using `pip install` often fails on non-x86/aarch64 platforms

Reply:
Appreciate the clarity! Could you please add a comment in the dockerfile clarifying this? It would be useful to document this for users interested in understanding or attempting to build without docker.